src/ts/InternalBitmovinYospacePlayer.ts

```diff
@@ -386,20 +390,35 @@ export class InternalBitmovinYospacePlayer implements BitmovinYospacePlayerAPI {
     return this.player.play(issuer);
   }
 
-  pause(issuer?: string): void {
-    if (this.playerPolicy.canPause()) {
-      this.player.pause();
-    } else {
-      this.handleYospacePolicyEvent(YospacePolicyErrorCode.PAUSE_NOT_ALLOWED);
+  private pullYospaceAdDataForLive(): Promise<void> {
+    if (
+      (this.yospaceSourceConfig.assetType !== YospaceAssetType.DVRLIVE && this.yospaceSourceConfig.assetType !== YospaceAssetType.LINEAR) ||
+      this.startSent
+    ) {
+      // Manually triggering the Yospace ad data update is only needed for Live and DVRLive sessions, and only the first time the stream is started.
+      return Promise.resolve();
     }
+
+    const adBreakUpdatePromise = new Promise<void>((resolve, reject) => {
```

Comment:
Nit naming: using `adBreak` feels wrong as it fetches everything I assume. Why is YoSpace calling it `Analytics` though.

```suggestion
    const adDataUpdatedPromise = new Promise<void>((resolve, reject) => {
```

Reply:
Fair point, will change. 

I don't know about the analytics naming 🤷‍♂️